client/src/Components/GameOver/GameOver.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components/macro';
```

Comment:
You could improve the import sequence by having first external imports (e.g. `styled components` and `react-router-dom`) then your components and lastly the service (`Context`)